rclpy/rclpy/node.py

```diff
@@ -31,6 +31,19 @@
 from rclpy.validate_topic_name import validate_topic_name
 
 
+def check_for_type_support(msg_type):
+    try:
+        if msg_type.__class__._TYPE_SUPPORT is None:
```

Comment:
Please only `try` / `except` the attribute access and not the invocation of `__import_type_support__`.